pkg/local_object_storage/metabase/last_resync_epoch_test.go

```diff
@@ -0,0 +1,55 @@
+package meta_test
+
+import (
+	"path/filepath"
+	"testing"
+
+	meta "github.com/nspcc-dev/neofs-node/pkg/local_object_storage/metabase"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDB_ReadLastResyncEpoch(t *testing.T) {
+	es := &epochState{
+		e: currEpoch,
+	}
+
+	db := meta.New([]meta.Option{
+		meta.WithPath(filepath.Join(t.TempDir(), "meta")),
+		meta.WithPermissions(0o600),
+		meta.WithEpochState(es),
+	}...)
+
+	require.NoError(t, db.Open(false))
+	require.NoError(t, db.Init())
+
+	t.Cleanup(func() {
+		db.Close()
+	})
+
+	checkEpoch := func(t *testing.T, epoch uint64) {
+		gotEpoch, err := db.ReadLastResyncEpoch()
+		require.NoError(t, err)
+		require.Equal(t, epoch, gotEpoch)
+		require.Equal(t, epoch, db.DumpInfo().LastResyncEpoch)
+	}
+
+	// Clean db without resynchronizations, so it is 0.
+	checkEpoch(t, 0)
+
+	// After Reset, last resync epoch == current epoch.
+	require.NoError(t, db.Reset())
```

Comment:
can we add a single non-zero test case, please?
1. empty base
2. empty to non-empty
3. non-empty to non-empty
4. non-empty reload with no changes

i gues that is the full list of use cases and we are ready to be confident